com.github.kagami.boram.yml

```
@@ -0,0 +1,243 @@
+app-id: com.github.kagami.boram
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: Boram
+finish-args:
+  # X11 + XShm access
+  - --share=ipc
+  - --socket=x11
```

```suggestion
  - --socket=fallback-x11
```

since it supports wayland